go.sum

```diff
@@ -122,10 +122,12 @@ github.com/kr/pretty v0.1.0/go.mod h1:dAy3ld7l9f0ibDNOQOHHMYYIIbhfbHSm3C4ZsoJORN
 github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
 github.com/lacework/conntrack v0.0.0-20200724052553-e16a10e68325/go.mod h1:yV2Weh5TLUf5Hhttml+lyLlwb9zIWUv3/ps0KnR8Utg=
+github.com/lacework/conntrack v0.0.0-20210812163929-c9e4ef49253e/go.mod h1:yV2Weh5TLUf5Hhttml+lyLlwb9zIWUv3/ps0KnR8Utg=
 github.com/lacework/go-avro v0.0.0-20170119191930-e2d751e5ad39/go.mod h1:qk0wZDxiuwrjZ78kE9RsqmsAIzP9WPRzU1rEE1nz/dk=
 github.com/lacework/go-metrics v0.0.0-20150907035206-330096136270/go.mod h1:vLI101WBSWOH2GQUkc78fkkfA9WGhBey1qLg8wstAX0=
 github.com/lacework/libaudit-go v0.0.0-20180727221555-a2e031d2f22c/go.mod h1:akSHYTrcKcUDeB7r1s+glBapPmU17v027clq/8qJstg=
 github.com/lacework/pcap v0.0.0-20201012204735-3a12bc5835a9/go.mod h1:yDJrEqmFT2k62qTIQ8k7dIeeLePeB4xlTIgs+Yo2Nbc=
+github.com/lacework/pcap v0.0.0-20210809204154-b716f8b3f08c/go.mod h1:yDJrEqmFT2k62qTIQ8k7dIeeLePeB4xlTIgs+Yo2Nbc=
```

Comment:
two entries for pcap?

Reply:
It seems to be there for other non-lacework packages too.